site/src/Librecores/ProjectRepoBundle/Resources/config/services.yml

```diff
@@ -1,89 +1,92 @@
+# Learn more about services, parameters and containers at
+# http://symfony.com/doc/current/book/service_container.html
+imports:
+    - { resource: legacy_aliases.yml }
+
+
 services:
     _defaults:
         autowire: true
         autoconfigure: true
+        public: false
+
+    # makes classes in src/AppBundle available to be used as services
+    # this creates a service per class whose id is the fully-qualified class name
+    Librecores\ProjectRepoBundle\:
+        resource: '../../../ProjectRepoBundle/*'
+        # you can exclude directories or files
+        # but if a service is unused, it's removed anyway
+        exclude: '../../../ProjectRepoBundle/{Entity,Repository,DataFixtures}'
+        # TODO: Evaluate why there is a Problem with class inside the folder DataFixtures.
```

Comment:
can you make this comment more specific? Other than that, excluding DataFixtures seems a reasonable thing to do.